communities.json

```diff
@@ -467,6 +467,13 @@
       "quote": "Discord allows Nuxt.js core team to chat and take decisions together in real-time. It engages our open source community to ask questions, get feedbacks for their projects and learn from each others.",
       "githubUrl": "https://github.com/nuxt/nuxt.js"
     },
+    {
+      "logo": "nextjs.svg",
+      "title": "Next.js",
+      "inviteCode": "yHzQeKFUcw",
```

Comment:
It looks like your server has a vanity invite code `nextjs` would you rather set it to that? Not sure if this invite code has any particular settings you want to keep, but non-vanity invite codes often have an expiration date

Reply:
Yeah, might as well use that I'd say 👍